### DISCUSSION TITLE
Show Prebuilds on Codespaces page

### DISCUSSION THREAD

**Discussion by NatoNathan** [community/community#47620] [Codespaces]
### Select Topic Area

Product Feedback

### Body

I like using Codespaces however I recently noticed that I was burning though my storage limit, even tho my usage to quitting low. turns out i had prebuilts enabled on one of my repos, but it took a while to find that repo. Can you please show a list of prebuilts on the Codespaces page to make it easier to identify them. and maybe an option to disable them on personal accounts?

**Comment by KartikM123** [comment]
Hi @NatoNathan! Thanks for the feedback! I'll forward this to the team 😄
